tensorboard/plugins/profile/pod_viewer/pod_viewer_dashboard/pod-viewer-dashboard.html

```diff
@@ -0,0 +1,124 @@
+<!--
+@license
+Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+
+<!--
+  pod-viewer is a view within the tf-profile-dashboard.
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../paper-slider/paper-slider.html">
+<link rel="import" href="details-card.html">
+<link rel="import" href="stack-bar-chart.html">
+<link rel="import" href="topology-graph.html">
+
+<dom-module id="pod-viewer-dashboard">
+  <template>
+<style>
+:host {
+  display: block;
+  margin-right: 1.5em;
+}
+#control {
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+  overflow:auto;
```

Comment:
as per code style guideline, please add a whitespace after colons here and below.

Reply:
Done.